src/services/identity/LaunchesService.ts

```diff
@@ -159,6 +161,41 @@ export class LaunchesService {
       throw siteIdResult.error
     }
 
+    try {
```

Comment:
Wrapped in a try catch since AmplifySDK will throw an (expected) not found error, which is ok